Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -1322,6 +1344,37 @@ public void onError(RequestError error) {
     return results;
   }
 
+  @Nullable
+  public GetCompletionDetailsResult completion_getSuggestionDetails(@NotNull final VirtualFile file,
+                                                                    final int id,
+                                                                    final String label,
+                                                                    final int offset) {
+    final String filePath = FileUtil.toSystemDependentName(file.getPath());
+    final Ref<GetCompletionDetailsResult> resultRef = new Ref<>();
+
+    final AnalysisServer server = myServer;
+    if (server == null) {
+      return null;
+    }
+
+    final CountDownLatch latch = new CountDownLatch(1);
+    server.completion_getSuggestionDetails(filePath, id, label, offset, new GetSuggestionDetailsConsumer() {
```

Comment:
I guess we need to use `getOriginalOffset()`. Offsets in the IntelliJ's model are normalized and may be different from the ones in the Analysis Serve's model.

Reply:
Good catch!